tools/nameid/nameid.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/letsencrypt/boulder/issuance"
+)
+
+func usage() {
+	fmt.Printf("Usage: %s [OPTIONS] [ISSUER CERTIFICATE(S)]\n", os.Args[0])
+}
+
+func main() {
+	var shorthandFlag = flag.Bool("s", false, "Display only the nameid for each given certificate")
+	flag.Parse()
+
+	if len(os.Args) <= 1 {
+		usage()
+		os.Exit(1)
+	}
+
+	for _, certFile := range flag.Args() {
+		issuer, err := issuance.LoadCertificate(certFile)
+		if err != nil {
+			if *shorthandFlag {
+				fmt.Println(err)
+			} else {
+				fmt.Printf("%s\t- %v", certFile, err)
+			}
```

Comment:
I don't think we should change the error format based on the `-s` flag. We should just unconditionally print the file name. And we shouldn't attempt to give it a tabular format. It should just be:

```suggestion
			fmt.Fprintf(os.Stderr, "%s: %s\n", certFile, err)
```

Also note the switch to `Fprintf(os.Stderr,` since errors should go to stderr.